unity/Assets/Scripts/Editor/Tests/Algorithms/Polygon/VisibilityTest.cs

```diff
@@ -245,6 +245,129 @@ public void AreaTest6()
             Assert.IsTrue(MathUtil.EqualsEps(expectedArea4, actualArea4));
         }
 
+        [Test]
+        public void AreaTest7()
+        {
+            var shape = LShape;
+
+            var vision1 = Visibility.Vision(shape, shape.Vertices.ElementAt(1));
+            Assert.AreEqual(shape.Vertices.Count, vision1.Vertices.Count);
+        }
+        
+        [Test]
+        public void AreaTest8()
+        {
+            var polygon = new Polygon2D(
+                new List<Vector2>()
+                {
+                    new Vector2(0, 0), // O
```

Comment:
What do the `O`'s mean?

Reply:
The `o` are form the original tests. I will remove them. Was for indication during debugging.